### PR TITLE
Conform README to core doc styleguide; Reword `Meetings` section

### PR DESCRIPTION
**Pull request by sagirk**
* Word-wrap at 80 characters
* Prefer affixing links to inlining links
* Fix broken links
* Reword text in meetings section to allow semantic links
* Remove unnecessary blank lines between sections

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,83 +1,98 @@
...
 ## Governance
 
 This initiative adopts the general Node.js [Code of Conduct], as well as its own
-[contributing guidelines].
+[Contributing Guidelines].
 
 ## Meetings
 
-Meeting notes can be accessed [here](./meetings). Future meetings can be listed using [this link](https://github.com/nodejs/website-redesign/issues?utf8=%E2%9C%93&q=is%3Aissue+is%3Aopen+Meeting+in%3Atitle).
+[Meeting notes] from past meetings are maintained for later reference.
```
Comment (maddhruv) [inline README.md]: why was the ref to the meetings dir and issues removed from here :thinking: ??

**Review by maddhruv — APPROVED** [review]
Awesome :tada: :tada:

File: README.md (hunk 1 above)
Comment (sagirk) [reply README.md]: Removed inline links in favour of affixed links — had to reword text in meetings section to fit this in.

As a bonus, this made the links more semantic and meaningful:
`[here]` => `[Meeting notes]`
`[this link]` => `[Query issues for "Meeting"]`

**Comment by sagirk** [comment]
@amiller-gh @chowdhurian Ping!

**Review by keywordnew — APPROVED** [review]
LGTM 👍🏽

Thanks for taking the initiative to improve the initiative's README, make it align better with the styleguide, and for opening a PR in this repo!

**Pull request merged**